.github/workflows/check-be-pull-request.yml

```diff
@@ -1,41 +1,43 @@
-name: Check Backend Pull Request
+name: "Check Backend Pull Request"
 
 on:
+  workflow_dispatch: # remove me
   pull_request:
-    types: [ opened, synchronize ]
+    types: [opened, synchronize]
     paths:
-      - 'backend/**'
-      - '!backend/README.md'
-      - '!backend/.*'
-      - 'backend/.eslintrc.js'
-
+      - "backend/**"
+      - "!backend/README.md"
+      - "!backend/.*"
+      - "backend/.eslintrc.js"
 
 jobs:
-
   check-be-pr:
     name: Check
     runs-on: ubuntu-latest
 
     steps:
-      -
-        name: ☁️ Checkout source
+      - name: ☁️ Checkout source
         uses: actions/checkout@v3
-      -
-        name: 🔧 Setup Node 16
+      - name: 🔧 Setup Node 16
         uses: actions/setup-node@v3
         with:
-          node-version: '16'
-          cache: 'npm'
+          node-version: "16"
+          cache: "npm"
           cache-dependency-path: backend/package-lock.json
-      -
-        name: 📦 Install dependencies
+      - name: 📦 Install dependencies
         run: npm ci --only-production --ignore-scripts
         working-directory: backend
-      # -
-      #   name: 🧪 Run tests
-      #   run: npm run test:ci
-      #   working-directory: backend
-      -
-        name: 🏗️ Run build
+      - name: 🧪 Run tests
+        run: npm run test:ci
+        working-directory: backend
+      - name: 📁 Upload test results
+        uses: actions/upload-artifact@v3
```

Comment:
Does this test result get deleted at some point?

Reply:
Ah, because GitHub cannot access files generated by other jobs/workflows. And the only way to "share" files is to upload them from one job, and download to another job/workflow.

Reply:
Will the upload override the release artifacts here? https://github.com/Infisical/infisical/releases/tag/v0.1.11 
Because I noticed that when i tried to upload artifacts, it would override

Reply:
So for each PR that is created, the artifacts of the test results will be uploaded in order to share the results with the user ?

Reply:
> Will the upload override the release artifacts here

The upload shouldn't override. They're artifacts of PR run.

> test results will be uploaded in order to share the results with the user

Yes